cornell-dti/course-plan
# [EZ Bug Fix] Close onboarding bug

### Summary 
When you haven't submitted your data for onboarding you shouldn't be able to close out of the onboarding modal.
My previous version of this fix wasn't right. I had to check the global onboardingData and not the local (not saved) fields from the onboarding form.
### Test Plan 
- Delete your firebase data & emulate onboarding for the first time
- Try to exit the onboarding modal without finishing
- Ensure that you can't
- After you submit your data, open up onboarding again and you should be able to exit out now.
